internal/runners/invite/invite.go

```diff
@@ -51,7 +51,7 @@ func (i *invite) Run(params *Params) error {
 	org := params.Org
 	if org.String() == "" {
 		if err := (&org).Set(i.project.Owner()); err != nil {
-			return locale.WrapInputError(err, "err_invite_org_current", "Could not use the owner of your current project.")
+			return err // do not wrap, as it overwrites the more helpful message displayed to the user (e.g. "You are not authenticated")
```

Comment:
This is what `errs.Wrap()` is for. Non-localized errors are not shown to the user. But we should almost always wrap our errors as it implicitly tells us the stack trace.

Also it's not true that it overwrites anything, if there are two localized errors in the chain then both are shown. If that's not what you were seeing then either we have some type of bug or the error down the chain isn't localized.

Reply:
Oh, this is actually a bug then: https://activestatef.atlassian.net/browse/DX-740

Reply:
Only one InputError is shown to the user, even if the error chain has multiple InputErrors. DX-740 mentions this. I've updated the PR to wrap errors, but to prefer any existing InputError for display to the user.